src/Symfony/Component/Console/Tests/Descriptor/ObjectsProvider.php

```diff
@@ -74,6 +74,10 @@ public static function getCommands()
 
     public static function getApplications()
     {
+        // The help text depends on the dynamic "PHP_SELF" value, which cannot
+        // be tested against. This makes the value predictable.
+        $_SERVER['PHP_SELF'] = 'app/console';
```

Comment:
Shouldn't we restore this, eg in tearDownAfterClass?

Reply:
Solved it in a nicer way :) (as there were now even more dynamic things)